include/TPP/Dialect/Transform/SCF/Dialect/Passes.h

```diff
@@ -0,0 +1,72 @@
+//===- Passes.h - Pass Entrypoints ------------------------------*- C++ -*-===//
```

Comment:
What's the point of this header? Don't we already have those passes in the pipeline?

Reply:
see https://github.com/alheinecke/tpp-mlir/commit/f97604c758323a6bd9285804bbfb7ddfbaf0d547 . I had issues compiling the PR, not sure why CI was able to do it, I got linker errors for tpp-run....

I guess this is only there for an include and a define. I moved things around in https://github.com/alheinecke/tpp-mlir/commit/f97604c758323a6bd9285804bbfb7ddfbaf0d547